sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,11 +1,24 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (2020-08-11)
+
+- Added the optional `serviceVersion` property to the `CertificateClient` optional parameters.
+    - It defaults to the latest supported API version, which currently is `7.1`.
+    - Other supported service version at the moment is `7.0`.
+    - If you were using `4.1.0-preview.1` version of this package, then this was previously named `apiVersion`.
+- `CertificateKeyType` now supports the `oct` type. To align with the Key Vault Keys client.
+- Added `recoverableDays` as an optional property to `CertificateProperties`.
+    - When creating and updating a certificate, set the `recoverableDays` property to define the number of days in which the certificate can be recovered after deletion. When certificates are read from the service, they will have the `recoverableDays` property in their `properties` field.
```

Comment:
> When certificates are read from the service, they will have the `recoverableDays` property in their `properties` field

I am not sure if "read from the service" makes it clear that we are talking about the `get` methods. I think we can skip it altogether assuming that anything that is set when creating/updating can be read when using the `get` methods.

```suggestion
    - When creating or updating a certificate, set the `recoverableDays` property to define the number of days in which the certificate can be recovered after deletion.
```

Reply:
Same for keys & secrets

Reply:
I decided to not go with this suggestion because this `create or update` section is invalid. This property is read-only. In the process of analyzing this, I made and closed this PR: https://github.com/Azure/azure-sdk-for-js/pull/10543 even though recoverableDays is part of the properties now, it's read only for all of the clients. Keys and Secrets don't extend the properties when creating entities (keys or secrets), only Certificates does this (but the service won't be reading it if it's sent on the `beginCreateCertificate` method either).